integration/token/fungible/dlog/dlog_test.go

```diff
@@ -94,16 +94,23 @@ var _ = Describe("EndToEnd", func() {
 			It("Malicious Transactions", Label("T9"), func() { fungible.TestMaliciousTransactions(ts.II, selector) })
 		})
 
-		Describe("T10 Fungible with Auditor ne Issuer and Endorsers", t.Label, func() {
+		Describe("Fungible with Auditor ne Issuer and Endorsers", t.Label, func() {
```

Comment:
we need to remove "ne"

Reply:
I think `ne` here stands for `not equal`. It should be fine.